testing/pom.xml

```diff
@@ -317,4 +251,102 @@
       </plugin>
     </plugins>
   </build>
+  <profiles>
+    <profile>
+      <id>it-cassandra-3.11</id>
```

Comment:
Can you also create an issue for getting the dev guide updated on how to run tests?

Reply:
#195